### DISCUSSION TITLE
Error: Package subpath "./utils" is not defined by "exports" in /node_modules/@vitest/runner/package.json

### DISCUSSION THREAD

**Discussion by kevinkim5** [vitest-dev/vitest#3869] [Q&A]
Hi all, seeking help with this issue while trying to use react testing library with vite: 
Error: Package subpath "./utils" is not defined by "exports" in /node_modules/@vitest/runner/package.json

@testing-library/dom: 9.3.1
@testing-library/jest-dom: 5.17.0
@testing-library/react: 14.0.0
jsdom: 22.1.0
vite: 3.2.7
vitest: 0.34.1
react: 18.2.0

setup.js and relevant settings have been updated in vite.config.js

**Comment by sheremet-va** [comment]
Check your lockfile. Looks like @vitest/runner is not the latest version

**Reply by kevinkim5** [reply]
@vitest/runner is v0.29.8

**Reply by kevinkim5** [reply]
Hi @sheremet-va , seems like the issue arises when using vitest together with eslint. Are there any workarounds apart from disabling eslint?

**Reply by sheremet-va** [reply]
update your vitest dependencies, they need to be the same version
